packages/presentation/lib/src/pages/home_page/bloc/home_bloc.dart

```diff
@@ -71,6 +71,8 @@ class _HomeBloc extends BlocImpl<BaseArguments, HomePageData>
     emit(data: tile, isLoading: isLoading);
     try {
       final trendingFull = await _fetchTrendingMovies();
+      if (trendingFull.isEmpty)
+        throw FetchDataException(details: 'Empty response');
```

Comment:
I don't think we should throw an exception here. empty state is not an exception, it's the same state as a list with 100 items. just emit the event, which will trigger empty state handling at the UI side